packages/headless/src/test/mock-result.ts

```diff
@@ -13,31 +13,18 @@ export function buildMockResult(config: Partial<Result> = {}): Result {
     summary: null,
     flags: '',
     hasHtmlVersion: false,
-    hasMobileHtmlVersion: false,
```

Comment:
I've cleaned up this interface with things that should be deprecated (`PascalCase` fields, `rating`, `htmlMobileVersion`), or that we do not actually support properly in headless as of today (all the `folding` related fields: parentChild, childResults).